<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogSettingsDialog</class>
 <widget class="QDialog" name="LogSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>336</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="nx::vms::client::desktop::Panel" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>64</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>64</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item>
       <widget class="QLabel" name="loggingLevelLabel">
        <property name="text">
         <string>Logging Level</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="loggingLevel"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>64</number>
      </property>
      <property name="topMargin">
       <number>16</number>
      </property>
      <property name="rightMargin">
       <number>64</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <property name="spacing">
       <number>8</number>
      </property>
      <item row="0" column="1">
       <widget class="QSpinBox" name="maxVolumeValue">
        <property name="maximum">
         <number>10240</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="splitBySizeUnits"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="splitBySizeLabel">
        <property name="text">
         <string>Split File by Size</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="splitByTimeLabel">
        <property name="text">
         <string>Split File by Time</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="splitByTimeValue">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="splitByTimeUnits"/>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="maxVolumeUnits"/>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="splitBySizeValue">
        <property name="maximum">
         <number>10240</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="splitByTimeChechBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="maxVolumeLabel">
        <property name="text">
         <string>Limit Max Volume</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="nx::vms::client::desktop::HintButton" name="maxVolumeHint" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="nx::vms::client::desktop::HintButton" name="splitBySizeHint" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="nx::vms::client::desktop::AlertBar" name="performanceWarning" native="true"/>
   </item>
   <item>
    <widget class="nx::vms::client::desktop::PromoBar" name="detailsWarning" native="true"/>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::HintButton</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/hint_button.h</header>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::Panel</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/panel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::PromoBar</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/control_bars.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::AlertBar</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/control_bars.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
